crypto/keyring/keyring.go

```diff
@@ -482,11 +482,11 @@ func (ks keystore) Delete(uid string) error {
 func (ks keystore) KeyByAddress(address sdk.Address) (*Record, error) {
 	ik, err := ks.db.Get(addrHexKeyAsString(address))
 	if err != nil {
-		return nil, wrapKeyNotFound(err, fmt.Sprint("key with address ", address.String(), "not found"))
```

Comment:
This is not related to that PR but I have noticed while testing that this formatting wasn't correct.